panel/chat/message.py

```diff
@@ -479,16 +479,28 @@ def _include_stylesheets_inplace(self, obj):
             for o in obj.objects:
                 self._include_stylesheets_inplace(o)
 
+    def _include_message_css_class_inplace(self, obj):
+        if hasattr(obj, "objects"):
+            for o in obj.objects:
+                self._include_message_css_class_inplace(o)
+        elif isinstance(obj, str):
+            return self._include_message_css_class_inplace(Markdown(obj))
```

Comment:
Don't quite understand this one, the `Markdown` object will never be used, will it?

Reply:
It's just converting it to markdown because str doesn't have attributes like `css_classes` and `sizing_mode` etc